test/python/bfloat16/test_conv2d.py

```diff
@@ -0,0 +1,99 @@
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge Conv2d operation test
+
+"""
+import pytest
+
+import tensorflow as tf
+import numpy as np
+import os
+
+import ngraph_bridge
+
+#Test Ngraph Op Convolution, TF Op:conv2d
+# Implemented based on NNP's unit test TEST(test_assign_layout, convolution_special_case)
+
+np.random.seed(5)
+
+# Colvolution Op is placed on NNP and conerted to bfloat16 only for the special case below, otherwise it falls back to CPU for compute
+# Check to assure:
+# The input rank is 4-D
+# The stride is less than the filter size
+# The Window and Data dilation is {1,1}
+# Filter shape is allowed
+# If any fail, then we should place Op on CPU for compute
+
+#Inputs
+N = 1
+H = 3
+W = 5
+C = 1
+
+filter = np.random.rand(1, 1, 1, 2)
```

Comment:
Please check if filter needs to be casted to bfloat

Reply:
thanks for pointing this out. yes, it does. changed it